### ISSUE TITLE
menu: fix state for alwaysopen=false + collapsibleMenu=false

### ISSUE THREAD

**Issue by ajeans** [McShelby/hugo-theme-relearn#498]
Hello

My site is configured with `alwaysopen` = `false` and , so that my menu is collapsed by default.

This works fine in version 5.11.2.

Upgrading to version 5.12.0 seems to break it, all my menu entries in `.topics` are now expanded.

config.json
```
{
  "baseURL": "/",
  "languageCode": "en-US",
  "defaultContentLanguage": "en",
  "title": "Quickign developer portal",
  "theme": [
    "quicksign",
    "hugo-theme-relearn"
  ],
  "metaDataFormat": "yaml",
  "defaultContentLanguageInSubdir": true,
  "params": {
    "description": "QuickSign developer portal",
    "author": "QuickSign",
    "alwaysopen": false,
    "disableBreadcrumb": false,
    "disableLandingPageButton": true,
    "titleSeparator": "::",
    "themeVariant": [ "quicksign", "neon", "relearn-light", "relearn-dark" ],
    "versions": {
      "applicant_js": "3.2.0",
      "caseflow": "0.37.0",
      "caseflow_js": "3.10.0",
      "caseflow_old": "0.36.0",
      "esig": "0.51.0",
      "esig_js": "3.6.0",
      "esig_old": "0.50.0",
      "front_apps": "2.7.1",
      "kyc": "0.34.0",
      "kyc_js": "3.4.0",
      "kyc_old": "0.33.0"
    }
  },
  "outputs": {
    "home": [
      "HTML", "SEARCH", "SEARCHPAGE"
    ]
  },
  "markup": {
    "goldmark": {
      "renderer": {
        "unsafe": true,
        "hardWraps": true
      }
    }
  },
  "Languages": {
    "en": {
      "title": "QuickSign Developer Portal",
      "weight": 1,
      "languageName": "English",
      "landingPageURL": "/en",
      "landingPageName": "<i class='fas fa-home'></i> Home",
      "menu": {
        "shortcuts": [
          {
            "name": "<i class='fab fa-fw fa-github'></i> GitHub repo",
            "identifier": "ds",
            "url": "https://github.com/Quicksign",
            "weight": 10
          },
          {
            "name": "<img src='/img/fa-quicksign-logo.png' /> QuickSign.com",
            "identifier": "website",
            "url": "https://www.quicksign.com/en/",
            "weight": 20
          }
        ]
      }
    },
    "fr": {
      "title": "Portail de développement QuickSign",
      "weight": 2,
      "languageName": "Français",
      "landingPageURL": "/fr",
      "landingPageName": "<i class='fas fa-home'></i> Accueil",
      "menu": {
        "shortcuts": [
          {
            "name": "<i class='fab fa-fw fa-github'></i> Repo GitHub",
            "identifier": "ds",
            "url": "https://github.com/Quicksign",
            "weight": 10
          },
          {
            "name": "<img src='/img/fa-quicksign-logo.png' /> QuickSign.com",
            "identifier": "website",
            "url": "https://www.quicksign.com/fr/",
            "weight": 20
          }
        ]
      }
    }
  }
}
```

5.11.2
![image](https://user-images.githubusercontent.com/2376384/222789078-aa3b89d9-a96c-42be-b39f-6970483452a8.png)

5.12.0 & 5.12.1
![image](https://user-images.githubusercontent.com/2376384/222788895-6f43ef59-d38b-4e99-ade4-062f615105cd.png)

**Comment by ajeans** [comment]
I looked at what changed in this area and c0a1c9cbcb7331f7e9b6bc4d2936e84b6ab7f06f seems to be the culprit.

Reverting it restores the behaviour.

**Comment by McShelby** [comment]
Thanks for reporting, this is fixed in 5.12.2

**Comment by ajeans** [comment]
I confirm this fixes the issue for me.

Thanks a lot for a very quick fix :)
